casanovo/casanovo.py

```
@@ -374,6 +391,131 @@ def train(
+            peak_mem_mb = 0
+
+            total_iters = warmup + n_iter
+            for i in range(total_iters):
+                is_timed = i >= warmup
+                if is_timed and torch.cuda.is_available():
+                    torch.cuda.reset_peak_memory_stats()
+
+                with tempfile.TemporaryDirectory() as tmp_dir:
+                    results_path = os.path.join(tmp_dir, "bench.mztab")
+                    with ModelRunner(
+                        cfg, model_path, output_path, None, False
```

Ruff (0.15.10)

[warning] Boolean positional value in function call

(FBT003)